Subscriber/TemplateSubscriber.php

```diff
@@ -55,4 +56,19 @@ public function addTemplateDir(Enlight_Event_EventArgs $args): void
 
         $controller->View()->addTemplateDir(dirname(__DIR__) . '/Resources/views');
     }
+
+    /**
+     * @return list<string>
+     */
+    public function onFilterJavascriptFiles(Enlight_Event_EventArgs $args): ?array
+    {
+        if (isset($this->pluginConfig['enableWebPJavascriptInFrontend']) && $this->pluginConfig['enableWebPJavascriptInFrontend'] === true) {
```

Comment:
Not sure if it should be like this, or 

```suggestion
        if (!isset($this->pluginConfig['enableWebPJavascriptInFrontend']) || $this->pluginConfig['enableWebPJavascriptInFrontend'] === true) {
```

for backward compatibility.

Reply:
Maybe just empty check?

Reply:
Since I read this https://www.beberlei.de/post/when_to_use_empty_in_php_i_say_never I tried to avoid empty. But I agree if the types are not really fixed, `empty` might still be reasonable. I am fine with all options.